response_operations_ui/templates/collection_exercise/ce-instrument-section.html

```diff
@@ -1,8 +1,71 @@
 <section class="ce-section {{'panel panel--simple panel--error' if missing_ci else ''}}">
+
+  {# 
+    This section needs to be updated once RAS-117 is completed.
+    Create new survey should provide an attribute, whether the survey is SEFT or EQ  
+  #}
+  {% set short_name = survey.shortName %}
```

Comment:
Feel free to disagree with me, but I think this should be done in the view code rather in the template.  I think the aim is to keep logic in the templates to a minimum where possible and it'd be easier to do something like 
```
{% if seft %}
....
```

Reply:
I will agree with you, but this code has been done just till the time we get Bilals PR in place. We can still take this logic to the view. I will try to put this logic to view code.